backend/docs/DATABASE.md

```diff
@@ -17,25 +17,30 @@ Refer to the CLI documentation [here](https://docs.microsoft.com/en-us/ef/core/m
 
 ## Database Migration Management
 
-The DB is setup and configured through Entity Framework Code-First processes. All dotnet ef commands must be run from the /dal directory.
+The database is setup and configured through Entity Framework Code-First processes. To use the Entity Framework CLI you will need to:
 
-To use the Entity Framework CLI you will need to install the **.NET SDK version**, **dotnet-ef tool** and add a `connectionstrings.json` configuration file to /dal. This connection string can also be included in a .env file in the /dal folder.
+- Install the **.NET SDK version** (download links below), 
+- Install **dotnet-ef tool** and,
+- Add a `connectionstrings.json` configuration file in the `/backend/dal` folder 
```

Comment:
The `connectionstrings.json` file is optional.  I’d reword to highlight this with “Optionally add”.

Reply:
Thanks for the suggestion, I can change it to "optionally" but first wanted to double check the use case with you:
- These instructions are for someone coming to the `/backend/dal/` folder to run migrations on their local (container) DB.

So in the above case - according to what I've read - when you run `dotnet ef database update`, the SDK will look for the `PimsContextFactory` that we created. Given that the `DAL` project is a class library it **doesn't** have an `appsettings.json` with a default connection string defined...

So, my question - if there's no `appsettings.json` and (as per instructions if we changed them above) you don't add either a `connectionstring.json` or `.env` file - where would `dotnet ef ...` find the connection information? 

How would `dotnet` know to go up one folder and then into `/api` to look for connection information?

Reply:
They would have to select one of the configuration options, but no single one is required.
- appsettings.config
- connectionstrings.config
- .env
- environment variables

Reply:
Good, I've modified the docs to account for your comments